joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/passes/frontend/XTypeHintCallLinker.scala

```diff
@@ -31,11 +31,11 @@ abstract class XTypeHintCallLinker(cpg: Cpg) extends CpgPass(cpg) {
   def calleeNames(c: Call): Seq[String] =
     (c.dynamicTypeHintFullName :+ c.typeFullName).filterNot(_.equals("ANY")).distinct
 
-  private def callees(names: Seq[String]): List[Method] = cpg.method.fullNameExact(names: _*).toList
+  def callees(names: Seq[String]): List[Method] = cpg.method.fullNameExact(names: _*).toList
 
   override def run(builder: DiffGraphBuilder): Unit = linkCalls(builder)
 
-  private def linkCalls(builder: DiffGraphBuilder): Unit = {
+  def linkCalls(builder: DiffGraphBuilder): Unit = {
```

Comment:
Can have a tighter scope
```suggestion
  protected def linkCalls(builder: DiffGraphBuilder): Unit = {
```

Reply:
Done